sdk/java-sdk-protobuf-testkit/src/main/java/kalix/javasdk/testkit/KalixProxyContainer.java

```diff
@@ -92,8 +93,10 @@ public void start() {
     Testcontainers.exposeHostPorts(googlePubSubPort);
     super.start();
     // Debug tooling: pass the Proxy logs into the client SLF4J
-    // Slf4jLogConsumer logConsumer = new Slf4jLogConsumer(LoggerFactory.getLogger("proxy-logs"));
-    // followOutput(logConsumer);
+    if ("true".equals(System.getenv("KALIX_TESTKIT_DEBUG"))) {
+      Slf4jLogConsumer logConsumer = new Slf4jLogConsumer(LoggerFactory.getLogger("proxy-logs"));
+      followOutput(logConsumer);
```

Comment:
Looks good, but I remember having done something similar in the past and struggling to get the output in a convenient format. 

If I'm not mistaken, I had to alter the docker to include another logback.xml with debug enabled. We also want the logs to not use json. The proxy has its own logback-dev-mode.xml, but we need to configure it explictly.

Reply:
Well, maybe changing the debug level for the proxy doesn't make much sense for the users as they won't be able to do much, only infer what might be wrong on their own application.